pkg/file/file.go

```diff
@@ -57,44 +66,200 @@ func NewFile(format string) (File, error) {
 	return nil, utils.NewErrInvalidSegment(format)
 }
 
-// CreateFile attempts to parse raw metro2 file contents
-func CreateFile(buf []byte) (File, error) {
```

Comment:
We need to keep this public API. It can call `NewFileFromReader(bytes.NewReader(buf))`.

Reply:
Yes